playbooks/byo/openshift-cluster/initialize_groups.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create initial host groups for localhost
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+  - always
+  tasks:
+  - include_vars: cluster_hosts.yml
+  - name: Evaluate group l_oo_all_hosts
+    add_host:
+      name: "{{ item }}"
+      groups: l_oo_all_hosts
```

Comment:
What's the significance of putting `l_` on this variable?  This is for my own sake.  I've seen this pattern used else where in this repo.

I generally shy away from naming variables with `l` as it appears to the viewer as `l`, `L`, `|`, or even `1`.